package.json

```
@@ -38,17 +38,16 @@
     "@essential-projects/tslint-config": "^1.1.3",
     "@types/bluebird-global": "^3.5.9",
     "@types/node": "^10.12.2",
-    "gulp": "^4.0.0",
-    "gulptraum": "^3.0.0",
-    "gulptraum-typescript": "^3.0.0",
-    "tsconfig": "^7.0.0",
-    "tslint": "^5.11.0",
-    "typescript": "^3.1.6"
+    "tslint": "^5.13.1",
+    "typescript": "^3.3.3333"
```

Why specific:

> "^3.3.3333"

seems a bit strange.

---

Can't unsee that, but it's real :D https://github.com/Microsoft/TypeScript/releases/tag/v3.3.3333

---

It is just the most current version.

The latest mandatory changes we need would be `^3.2`, but I don't see a problem with using the latest version instead.

---

Why then not just stick with 

```suggestion
    "typescript": "^3.2.0"
```

?

This is a`^` dependency anyway, so it doesn't really matter what we put in `minor` and `patch` anyway.

---

If it doesn't matter then why not leave it at the latest version?

---

I'm cool with that too. I just figured it may help to solve the confusion @cmg-dev felt. 😉